server/database.ts

```diff
@@ -29,6 +29,11 @@ export function initDB(dbFile: string): Database {
             Timestamp INTEGER, 
             FOREIGN KEY(Username) REFERENCES Users(Username)
         );`);
+
+        newDB.run(`CREATE TABLE if not exists "PostDislikes" (
+            ID TEXT PRIMARY KEY, 
+            Username TEXT
```

Comment:
ID and Username should be foreign keys to Post and User respectively. Thus ID and Username are Partial Primary Keys.